nilearn/input_data/multi_nifti_masker.py

```diff
@@ -201,6 +201,7 @@ def fit(self, imgs=None, y=None):
         # Resampling: allows the user to change the affine, the shape or both.
         if self.verbose > 0:
             print("[%s.transform] Resampling mask" % self.__class__.__name__)
+        print(hash(self.mask_img_))
```

Comment:
Commits are labelled "WIP". I renamed the PR itself do that we don't review unfinished code.